CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,140 @@
+
+# Code of Conduct
+
+## Our Pledge
+
+We as contributors pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
```

Comment:
Bitcoin Core is not a community, and this paragraph seems to suggest discrimination on the basis of personal beliefs or actions outside the project.

Instead, IF there's a CoC, it should be limited to project-specific forums like GitHub, #bitcoin-core-dev IRC meetings (maybe channel, though it occasionally strays from the topic), and such. Things like in-person meetings are tricky since there isn't always a clear delineation between personal and professional activities.